AMW_angular/io/src/app/deployment/deployments-list.component.ts

```diff
@@ -176,7 +174,7 @@ export class DeploymentsListComponent {
   }
 
   logViewerLink(deploymentId: number){
-    window.location.href = '/AMW_web/pages/logView.xhtml?deploymentId=' + deploymentId + '&filters=' + encodeURI(JSON.stringify(this.filtersInUrl))
```

Comment:
I just added this, can't we keep it? See comment here: https://github.com/liimaorg/liima/pull/247#issuecomment-344621199